src/infinity-menu.js

```diff
@@ -31,7 +32,23 @@ export default class InfinityMenu extends React.Component {
 		event.preventDefault();
 		if (!this.state.search.isSearching || !this.state.search.searchInput.length) {
 			node.isOpen = !node.isOpen;
-			NestedObjects.set(tree, keyPath, node);
```

Comment:
i was playing around and forgot about it.  i'll get it back in as i work through my mess